generator.js

```diff
@@ -43,7 +43,7 @@ function formatName(string) {
 function componentTemplate(name, svg) {
   let component = "import React from 'react'\n"
   component += "import { IconProps } from './types'\n"
-  component += `const ${name}: React.FC<IconProps> = (props) => (`
+  component += `const ${name}: React.FC<IconProps> = (props: IconProps):any => (`
```

Comment:
Are you sure that this is required? @mmalfertheiner do we need here a return value?

Reply:
Simon, React.FC is an interface, which defines the parameters and the return value. Thats why the original version was correct.